perfkitbenchmarker/providers/openstack/os_disk.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, disk_spec, name, zone, image=None):
         self.name = name
         self.zone = zone
         self.device = ""
-        self.virtual_disks = (c for c in "cdefghijklmnopqrstuvwxyz")
```

Comment:
This was causing a PicklingError when the spec was being pickled. The error was caused by the generator.